<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
       name="ur10" >

  <!-- common stuff -->
  <!-- <xacro:include filename="$(find ur_description)/urdf/common.gazebo.xacro" /> -->
  <xacro:include filename="$(find osrf_gear)/vendor/ur_description/urdf/common.gazebo.xacro" />

  <!-- ur10 -->
  <!-- <xacro:include filename="$(find ur_description)/urdf/ur10.urdf.xacro" /> -->
  <xacro:include filename="$(find osrf_gear)/vendor/ur_description/urdf/ur10.urdf.xacro" />

  <!-- arm -->
  <xacro:ur10_robot prefix="" />

</robot>
